yt_dlp/extractor/youtube.py

```diff
@@ -3534,7 +3559,7 @@ def _real_extract(self, url):
             ie=YoutubeTabIE.ie_key())
 
 
-class YoutubeSearchIE(SearchInfoExtractor, YoutubeBaseInfoExtractor):
+class YoutubeSearchIE(SearchInfoExtractor, YoutubeTabIE):
```

Comment:
Wasn't quite sure what we should do here to let YoutubeSearchIE access extract_response.

Reply:
this is fine. Another way would be to move `extract_response` and related functions to the `BaseIE`